demo-dep-output.yaml

```diff
@@ -3,1196 +3,1175 @@
   dependencies:
   - name: github.com/go-logr/logr
     version: v1.2.0
-    type: ""
-    indirect: false
-    resolvedidentifier: ""
-    extras: {}
+    labels:
+    - konveyor.io/dep-source=downloadable
```

Comment:
I dont think I understand where 'downloadable' comes from, shouldn't it be 'open-source'?

Reply:
I was making it up for golang. I was considering that something from go.mod, meant that it can be downloaded, while if there was something in vendor, then it would be "vendored". IDK thoughts?